### ISSUE TITLE
Does LoRA attach to attention modules here?

### ISSUE THREAD

**Issue by OliverXUZY** [KohakuBlueleaf/LyCORIS#181]
Hello!
  Thank you for your interesting work! 

I tried to use this to finetune some transformer block with attention modules. I found in script [here](https://github.com/KohakuBlueleaf/LyCORIS/blob/4cd782c819738985853c076ade197f9cd5fc48c6/lycoris/wrapper.py#L291-L305), when creating single lora module, it will return `None` if module name module name is not `Linear` or `Conv2d`. I am wondering whether it will apply to other modules like `nn.MultiheadAttention`.

Thank you so much!

**Comment by KohakuBlueleaf** [comment]
nn.MultiheadAttention is super tricky since it combined 3 input proj layer together (unless the dim is different)

For now I'm busy in reconstruct whole library API and currently no plan to support pytorch's MHA module

If you have any idea how to support it efficiently (make 4 lyco module for MHA and when forward, rebuild the input proj from 3 of them and rebuild the output proj from the last)
You can open PR for it (remember to open PR to Dev branch)
